dev-server/index.js

```diff
@@ -0,0 +1,8 @@
+const express = require('express');
+const path = require('path');
+
+const app = express();
+
+app.use(express.static(path.join(__dirname, '../dist')));

```

Comment:
there is simple node static server for this purpose

Reply:
done